CMakeLists.txt

```diff
@@ -94,30 +94,29 @@ if(APPLE)
   # RPATH stuff
   set(CMAKE_MACOSX_RPATH ON)
   if(NOT IOS)
-    # Determine if we can link against ML Compute
-    set(MLCOMPUTE_FOUND OFF)
+    # Determine if we can link against MPSGraph
+    set(MPS_FOUND OFF)
     execute_process(
       COMMAND bash -c "xcrun --sdk macosx --show-sdk-path"
       OUTPUT_VARIABLE _macosx_sdk_path
       OUTPUT_STRIP_TRAILING_WHITESPACE)
-
+    set(_MPS_supported_os_version OFF)
+    if (CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin"
+        AND DARWIN_MAJOR_VERSION VERSION_GREATER_EQUAL 21
```

Comment:
21 or 12?

Reply:
This is the DARWIN_MAJOR_VERSION which are different than the OSX versions. If you do `uname -a` you can see the darwin kernel version.